docs/faq.rst

```diff
@@ -702,6 +702,23 @@ Now Borg will be bandwidth limited. Nice thing about pv is that you can change r
 .. _pipeviewer: http://www.ivarch.com/programs/pv.shtml
 
 
+How can I avoid unwanted base directories getting stored into archives?
+-----------------------------------------------------------------------
+
+Possible use cases:
+
+- Another file system is mounted and you want to backup it with original paths.
+- You have created a BTRFS snapshot in a ``/.snapshots`` directory for backup.
+
+To achieve this, run ``borg_create`` within the mountpoint/snapshot directory:
```

Comment:
Hmm, how is this rendered? Is this a link to the borg create docs?

Or should it rather be ```borg create``` (with blank, not underscore)?

Reply:
That should be a space, not underscore. Fixed.